qutebrowser/mainwindow/statusbar/bar.py

```diff
@@ -257,6 +252,23 @@ def _draw_widgets(self):
                 self.prog.enabled = True
                 if tab:
                     self.prog.on_tab_changed(tab)
+            elif segment.startswith('text'):
+                cur_widget = textbase.TextBase()
+                self.text_widgets.append(cur_widget)
+                cur_widget.setText(segment.partition(':')[2])
```

Comment:
Maybe

```suggestion
                cur_widget.setText(segment.split(':', maxsplit=1)[1])
```

would be clearer? No hard feelings either way.